apps/console/package.json

```diff
@@ -81,6 +81,7 @@
         "@wso2is/admin.remote-userstores.v1": "^1.7.45",
         "@wso2is/admin.roles.v1": "^2.25.89",
         "@wso2is/admin.roles.v2": "^2.27.33",
+        "@wso2is/admin.rules.v1": "workspace:^",
```

Comment:
Shall we use the exact version to be consistent with the current practice?

Reply:
Yes, this is updated.